torch/csrc/jit/codegen/cuda/scheduler/pointwise.cpp

```diff
@@ -467,6 +600,25 @@ c10::optional<PointwiseParams> getPointwiseHeuristics(
   params.break_point = break_point;
   params.split_block = bdimy > 1;
 
+  auto expanded_vector_word_size = expandVectorizationToContigMergedDomains(
```

Comment:
Can  you move the other vectorize logic down, and move this logic just above where we start filling entries in params. I try to leave the params filling to the end of the heuristics functions as much as I can. Logic looks good to me, just looking to try and keep the vectorize logic as close together as possible in the function.